__tests__/app.test.js

```diff
@@ -256,5 +256,41 @@ describe('endpoint GET /api/users', ()=>{
             expect(body.users).toEqual(testData.userData)
         })
     })
+})
 
+describe('endpoint GET /api/articles (topic query)', ()=>{
+    test('receives 200 response and responds with articles filtered by the topic value specified in the query', ()=>{
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({body})=>{
+            expect(body.articles).toEqual( [{
```

Comment:
This is a good test but as an optional refactor you could use a flexible jest matcher such as toMatchObject